Create ui-c8y-1022-46-4-maintenance-mode-not-reflected-correctly-in-left-navigation-bar.md (#3886)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-46-4-maintenance-mode-not-reflected-correctly-in-left-navigation-bar.md

---
date: ""

> title: Maintenance mode not reflected correctly in left navigation bar

Reviewer: ```suggestion
title: Device status changes were not immediately reflected in Navigator
```

> title: Device status changes were not immediately reflected in Navigator

Reviewer 2: ```suggestion
title: Device status changes now immediately reflected in the navigator
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4994
version: 1022.46.4
---

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer: ```suggestion
When a device was put into maintenance mode (e.g. from the **Device status** widget), or vice versa, this status change was not immediately reflected in the left navigation bar. With this change, the left navigation bar now instantly displays the new device status. This change ensures that users have clear visibility into the current operational state of their devices.
```

> When a device was put into maintenance mode (e.g. from the **Device status** widget), or vice versa, this status change was not immediately reflected in the left navigation bar. With this change, the left navigation bar now instantly displays the new device status. This change ensures that users have clear visibility into the current operational state of their devices.

Reviewer 2: ```suggestion
When the status of a device was changed (for example, from the "Device status" widget), the status change was not immediately reflected in the navigator. With this change, the navigator now instantly displays the new device status. This change ensures that users have clear visibility into the current operational state of their devices.
```
